.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,46 @@
+name: GitHub Actions Demo
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on:
+  pull_request:
+    branches:    
+      - master
+  push:
+    branches:
+      - master
+    
+jobs:
+  pr_and_merge_ci:
+    runs-on: ubuntu-latest
+    container:
+      image: probrian35/vectorcxx_dev:latest
```

Comment:
User personal build for temp, will migrate to our repo later

Reply:
As above, we could try to pin the image to a specific version to make the build reproducible